src/components/Dashboard/Searchbar.tsx

```diff
@@ -4,32 +4,40 @@ import { splitNSearch } from '@/utils/splitNSearch';
 
 function Searchbar({ label }: searchProps) {
     const [query, setQuery] = useState('');
+    let btnStyle = styles.btnActive;
+    let btnDisabled = false;
 
     const handleKeyPress = (event: KeyboardEvent) => {
         if (event.key === 'Enter') {
             splitNSearch(query);
         }
     };
 
+    if (query === '') {
+        btnStyle = styles.btnDisabled;
+        btnDisabled = true;
+    }
+
     return (
         <section className={styles.container}>
-            <label htmlFor="search" className={styles.searchLabel}>
-                {label}
-            </label>
             <input
+                data-testid="searchbar_input"
                 type="text"
                 id="search"
                 value={query}
                 onChange={(e) => setQuery(e.target.value)}
                 onKeyDown={(e) => handleKeyPress(e)}
                 className={styles.searchBar}
+                placeholder={label + ':'}
             />
             <button
```

Comment:
Please change the cursor to a hand pointer on hover over this button

Reply:
done